os/arch/arm/src/amebasmart/amebasmart_mipi.c

```diff
@@ -310,9 +312,16 @@ static void amebasmart_mipidsi_isr(void)
 		amebasmart_mipidsi_rcmd_decode(MIPIx, 2);
 	}
 
-	if (reg_val & (MIPI_BIT_RCMD1 | MIPI_BIT_RCMD2 | MIPI_BIT_RCMD3)){
-		mipillvdbg("RCMD: %x %x %x %x %x\n", rx_data_buff[0], rx_data_buff[1], rx_data_buff[2], rx_data_buff[3], rx_data_buff[4]);
-	}
+ 	if (reg_val & (MIPI_BIT_RCMD1 | MIPI_BIT_RCMD2 | MIPI_BIT_RCMD3)){
+		if (rx_data_ptr) {
+			mipillvdbg("RCMD " );
+			for (int i = 0; i < rx_data_len; i ++) {
+				mipillvdbg("%x ", rx_data_ptr[i]);
+			}
+			mipillvdbg("\n" );
+		}
+		
+ 	}
```

Comment:
```suggestion
	if (reg_val & (MIPI_BIT_RCMD1 | MIPI_BIT_RCMD2 | MIPI_BIT_RCMD3)) {
		if (rx_data_ptr) {
			mipillvdbg("RCMD ");
			for (int i = 0; i < rx_data_len; i++) {
				mipillvdbg("%x ", rx_data_ptr[i]);
			}
			mipillvdbg("\n");
		}
	}
```
Remove unnecessary spaces.